services/src/main/java/org/keycloak/validation/DefaultClientValidationProvider.java

```diff
@@ -264,4 +269,20 @@ private void validateJwks(ValidationContext<ClientModel> context) {
             context.addError("jwksUrl", "Illegal to use both jwks_uri and jwks_string", "duplicatedJwksSettings");
         }
     }
+
+    private void validateDefaultAcrValues(ValidationContext<ClientModel> context) {
+        ClientModel client = context.getObjectToValidate();
+        List<String> defaultAcrValues = AcrUtils.getDefaultAcrValues(client);
+        for (String configuredAcr : defaultAcrValues) {
+            Map<String, Integer> acrToLoaMap = AcrUtils.getAcrLoaMap(client);
```

Comment:
This method might works well if we move L277-280 to just before L276.

Reply:
Good catch! I've updated it as you suggested as that is more effective.